include/swift/AST/SemanticAttrs.def

```diff
@@ -89,6 +89,7 @@ SEMANTICS_ATTR(FASTPATH, "fastpath")
 SEMANTICS_ATTR(SLOWPATH, "slowpath")
 SEMANTICS_ATTR(PROGRAMTERMINATION_POINT, "programtermination_point")
 SEMANTICS_ATTR(CONVERT_TO_OBJECTIVE_C, "convertToObjectiveC")
+SEMANTICS_ATTR(MUST_TAIL, "musttail")
```

Comment:
Can you make a more descriptive name perhaps? Maybe give it a prefix? Not sure. To be clear, I am just talking about the "musttail" part that appears in the source. I think MUST_TAIL looks perfect on the c++ side!